packages/contentful-apps/redirects/DESIGN.md

```diff
@@ -0,0 +1,68 @@
+# Redirects app Design
```

Comment:
I am not sure if it makes sense to have this document in here as it is.
Maybe we should transform it into something like a ToDo List or a Roadmap?
So only the open tasks or possible improvements would be listed.
The objective can be moved to the readme.
Parts from the document could also be transferred to a document that describes how the app should be used and what it produces.
We should talk about that.